src/core/translations/dutch-translations.constant.js

```diff
@@ -329,6 +329,7 @@ angular.module('udb.core')
       'contactPoint': 'Contactinfo',
       'status': 'Status',
       'bookingAvailability': 'Tickets & plaatsen',
+      'attendance': 'Aanwezigheidsvorm (fysiek / online)',
```

Comment:
@corneelwille is "Aanwezigheidsvorm" the copy that you decided on? I remember you asking for feedback about how to translate it in Slack a while back and that you were going to check with marcom.

Reply:
Also see the proposal of adding "Beide" https://github.com/cultuurnet/udb3-angular/pull/714#discussion_r870369168